Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  HKActivitySummaryExtensions.swift
+//  SwifterSwift
+//
+//  Created by Gilbert Lo on 7/3/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(HealthKit)
+import HealthKit
+
+// MARK: - Properties
+@available(watchOS 2.2, *)
+public extension HKActivitySummary {
+
+    /// SwifterSwift: Check if stand goal is met
+    var isStandGoalMet: Bool {
+        let unit = HKUnit.count()
+        let standHours = appleStandHours.doubleValue(for: unit)
```

Comment:
Is there a way to compare this without using an arbitrary unit? 
I don't know the API, so these are only questions :)

Reply:
Unfortunately no =[

Reply:
Sorry, I just did doing more research on this, turns out there is a way to compare this without using an arbitrary unit. Just push the update